examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -618,7 +618,7 @@ bool RvcDevice::GetSupportedMapById(uint8_t aMapId, uint32_t & listIndex, MapStr
 
     while (listIndex < listSize)
     {
-        if (mSupportedMaps[listIndex].mapId == aMapId)
+        if (mSupportedMaps[listIndex].mapID == aMapId)
```

Comment:
If you have to change Id vs ID, shouldn't aMapId also go to aMapID?  Once you start polishing, it never stops.

Reply:
I changed these. Not sure when...